bin/iron/src/app.rs

```diff
@@ -59,7 +59,8 @@ impl IronApp {
                 iron_forge::commands::foundry_get_abi,
                 iron_rpc::commands::rpc_send_transaction,
                 iron_connections::commands::connections_affinity_for,
-                iron_connections::commands::connections_set_affinity
+                iron_connections::commands::connections_set_affinity,
+                iron_sync::commands::alchemy_supported_network
```

Comment:
can you rename to `sync_alchemy_supported_network`? to keep the convention of starting the command name with same name of the crate it's in